tbx/static_src/javascript/components/mode-switcher.js

```diff
@@ -25,15 +25,10 @@ class ModeSwitcher {
     }
 
     bindEventListeners() {
-        this.modeRadios.forEach((radio) => {
-            radio.addEventListener('change', () => {
-                if (radio.checked) {
-                    const mode = radio.value;
-                    if (mode === this.mode) return;
-                    this.mode = mode;
-                    this.toggleMode();
-                }
-            });
+        this.modeButton.addEventListener('click', (event) => {
+            event.preventDefault();
+            this.mode = this.mode === 'dark' ? 'light' : 'dark';
```

Comment:
Suggestion - could add some JS in here to give the aria-label some context. It would need updating in the template as well.
```suggestion
            this.mode = this.mode === 'dark' ? 'light' : 'dark';
                        this.modeButton.setAttribute(
                'aria-label',
                this.mode === 'dark'
                    ? 'Switch to light mode'
                    : 'Switch to dark mode',
            );
```

Reply:
Good idea - I've implemented this now.